creation.rst

```diff
@@ -0,0 +1,60 @@
+Creating your wallet
+====================
+
+Choose a name
+-------------
+
+Some people use multiple wallets!  Just make sure it's very clear to avoid confusion!
+
+.. image:: png/creation_namewallet.png
+
+Choose the wallet type
+----------------------
+
+Electrum supports some advanced features.  
+Let's stick with the default.
+
+.. image:: png/creation_wallettype.png
+
+
+Choose how keys are stored
+--------------------------
+
+Electrum supports some advanced features.
+Let's stick with the default.
+
+.. image:: png/creation_keystore.png
+
+
+Write down your seed
+--------------------
+
+These words can be used to recover your wallet and spend your bitcoin.
+Make sure you write these down, make backups, and have multiple copies.
+
+DO NOT SHARE.  KEEP YOURS SAFE.
+
+.. image:: png/creation_newseed.png
+
+
+Confirm your Seed
+-----------------
+
+Enter in your seed words to double check you have actually backed them up.
+
+.. image:: png/creation_confirmseed.png
+
+
+You may encrypt your wallet
+---------------------------
+
+You may enter a password here if you want to encrypt your wallet on your computer.
+Be careful.  This will not apply to your seed words you wrote down.
```

Comment:
I don't expect laymen to understand how it works.  I don't want them to think the seed is safe to leave around because they "added a password"